Buddies.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.1.32104.313
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Buddies.API", "Buddies.API\Buddies.API.csproj", "{F32ADFB2-8C9B-4D57-A58A-A8ED5D3FEB72}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Buddies.Tests", "Buddies.Tests\Buddies.Tests.csproj", "{5658F86D-3821-428E-8889-258CE6149F1B}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{F32ADFB2-8C9B-4D57-A58A-A8ED5D3FEB72}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{F32ADFB2-8C9B-4D57-A58A-A8ED5D3FEB72}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{F32ADFB2-8C9B-4D57-A58A-A8ED5D3FEB72}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{F32ADFB2-8C9B-4D57-A58A-A8ED5D3FEB72}.Release|Any CPU.Build.0 = Release|Any CPU
+		{5658F86D-3821-428E-8889-258CE6149F1B}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{5658F86D-3821-428E-8889-258CE6149F1B}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{5658F86D-3821-428E-8889-258CE6149F1B}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{5658F86D-3821-428E-8889-258CE6149F1B}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {2FBE3108-3515-4CD8-B62A-4930EBEC747E}
+	EndGlobalSection
+EndGlobal
```

Comment:
I'm not too familiar with Visual Studio, is it best practice to check this file in?

Reply:
Yes it is best practice to check this file in, the `.gitignore` file at the root of the repository was generated through the .NET SDK and already excludes all unnecessary files from Visual Studio/.NET development.